backend/src/main/java/trkpo/spbstu/hospitalavailability/model/Client.java

```diff
@@ -0,0 +1,21 @@
+package trkpo.spbstu.hospitalavailability.model;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "client")
+@NoArgsConstructor
+public class Client {
+    @Id
+    @GeneratedValue
```

Comment:
Я бы наверное явно поставил уточнение, чтобы он юзал генерацию бд постгресса, а не выдумывал там что-то с сиквенсами и тд

```suggestion
    @GeneratedValue(strategy = GenerationType.IDENTITY)
```